drivers/rtc/rtc_ll_stm32.c

```diff
@@ -123,14 +95,14 @@ static int rtc_stm32_configure(const struct device *dev)
 	if ((hour_format != LL_RTC_HOURFORMAT_24HOUR) ||
 	    (sync_prescaler != cfg->sync_prescaler) ||
 	    (async_prescaler != cfg->async_prescaler)) {
-		err = rtc_stm32_enter_initialization_mode(false);
```

Comment:
The error return type is not the same as for rtc_stm32_enter_initialization_mode. You should use an ErrorStatus and check if it's different from SUCCESS and set err accordingly. (same for the other call at line 194).

Reply:
I fixed this issue. Does it look correct now?